src/Activity.js

```diff
@@ -63,7 +60,7 @@ class Activity {
 
   userDataForWeek(releventData) {
     return this.userRepo.getWeekFromDate(this.date, this.user.id, this.activityData).map((data) => {
-      return {x: data.date, y: data[releventData]}
+      return {[data.date]: data[releventData]};
```

Comment:
I like this change. I think it makes more sense to return the data in this format from our method and then change it to the graph display format in scripts